xds/src/main/java/io/grpc/xds/internal/certprovider/CertificateProviderRegistry.java

```diff
@@ -38,6 +38,14 @@ public CertificateProviderRegistry() {
   public static synchronized CertificateProviderRegistry getInstance() {
     if (instance == null) {
       instance = new CertificateProviderRegistry();
+      // TODO(sanjaypujare): replace with Java's SPI mechanism and META-INF resource
+      CertificateProviderProvider certificateProviderProvider =
+          new FileWatcherCertificateProviderProvider();
+      instance.register(certificateProviderProvider);
```

Comment:
nit: Combine with the previous line (avoiding the local variable)? It seems the line wouldn't even need wrapping. Ditto below.